EIPS/eip-5851.md

```diff
@@ -0,0 +1,256 @@
+    * 
+    * @param title defines the name of the claim field
+    * @param kind is the nature of the data (bool,string,address,bytes,..)
+    * @param description additional information about claim details.
+     */
+    struct Metadata {
+        string title;
+        string kind;
+        string description;
+    }
+```
+
+### 2. Values Information
+
+This structure will be used to define the actual claim information, based on the description of the `Metadata` structure, the structure is similar to `Values` structure of [EIP-3475](./eip-3475.md).
+
+### 3. Claim structure
+
+Claims (eg. `age >= 18`, jurisdiction in allowlist, etc.) are represented by one or many instances of the `Claim` structure below:
+
+```solidity
+    /** Claims
+    * 
+    * Claims structure consist of the conditions and value that holder claims to associate and verifier has to validate them.
+    * @notice the below given parameters are for reference purposes only, developers can optimize the fields that are needed to be represented on-chain by using schemes like TLV, encoding into base64 etc.
+    * @dev structure that defines the parameters for specific claims of the SBT certificate
+    * @notice this structure is used for the verification process, it contains the metadata, logic and expectation
+    * @notice logic can represent either the enum format for defining the different operations, or they can be logic operators (stored in form of ASCII figure based on unicode standard). like  e.g: 
+("⊄" = U+2284, "⊂" = U+2282,  "<" = U+003C , "<=" = U + 2265,"==" = U + 003D, "!="U + 2260, ">=" = U + 2265,">" =  U + 2262).
+    */
+    struct Claim {
+        Metadata metadata;
+        string logic;
+        Values expectation;
+   
+    }
+```
+
+description of some logic functions that can be used are as follows: 
+
+| Symbol | Description |
+|--------|--------------|
+| ⊄ | does not belong to the set of values (or range) defined by the corresponding `Values`  |
+| ⊂ | condition that the parameter belongs to one of values defined by the `Values`  |
+| < | condition that the parameter is greater than  value defined by the `Values`  |
+| == | condition that the parameter is strictly equal to the value defined by the `Values` structure |
+
+#### Example
+
+```json
+{
+   "title":"age",
+   "type":"unit",
+   "description":"age of the person based on the birth date on the legal document",
+   "logic":">=",
+   "value":"18"
+}
+```
+
+Defines the condition encoded for the identity index 1 (i.e the holder must be equal or more than 18 years old).
+
+### Interface specification
+
+#### Verifier
+
+```solidity
+
+    /// @notice getter function to validate if the address `claimer` is the holder of the claim Defined by the tokenId `SBTID`
+    /// @dev it MUST be Defining the logic to fetch the result of the ownership verification of NFT(with metadata generated using ZK schemes).
+    /// @dev logic given here MUST be one of ("⊄", "⊂", "<", "<=", "==", "!=", ">=", ">")
```

Comment:
If `logic` is something other than one of these options, what happens? The documentation for the `Claim` struct allows for other values

Reply:
These are basic relational logic for minimal KYC constraint usage, more complex logic can be defined by multiple claims.